src/main/res/values/strings.xml

```diff
@@ -1385,7 +1385,7 @@
  May be transliterated. -->
     <string name="card_name_ravkav">Rav-Kav</string>
     <!-- Translators: appears as an agency if user used official top-up app. -->
-    <string name="ravkav_agency_topup_app">RavKav Top-up app</string>
+    <string name="ravkav_agency_topup_app">RavKav HopOn app</string>
 
```

Comment:
https://www.youtube.com/watch?v=_CN6TlA_KoI
hopon.co
https://play.google.com/store/apps/details?id=co.hopon.client&hl=en_US
Not sure if i am mixing things, which is the HopOn, or which is the TopUp,  but nothing is "Top-up"

Reply:
I had a dig into this too, because it's strange that there are **two** apps... and even [a government website](https://www.gov.il/en/departments/guides/multi_line_card?chapterIndex=5) mentions both...

* [Rav-Kav Online](https://ravkavonline.co.il/en/) ([Play Store](https://play.google.com/store/apps/details?id=com.pcentra.ravkavonlinemobile))

  Their Play Store listing and website both say `Rav-Kav` consistently.

* [HopOn](https://hopon.co.il/) ([Play Store](https://play.google.com/store/apps/details?id=co.hopon.client)) (to view the English version, click `עברית` at the top-left, then press `English`)

  The Play Store listing says `RavKav` in the title, but both `Rav Kav` and `Rav-Kav` in the description. Their website mostly says `Rav-Kav`.

* [The Israel government website](https://www.gov.il/en/departments/guides/multi_line_card?chapterIndex=1) says `Rav-Kav` all the time.

So I think we can settle for at least `Rav-Kav` all the time.

The actual implementation here mentions a specific agency ID `0x19`.  If we're lucky they will be two different things, but maybe @phcoder's source knows.

Reply:
This was meant only as an "app for top-up", not the official name of the app. I don't remember which app I used to do the test top-up